src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/OpenObject.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections.Generic;
+
+namespace CoreWCF.IdentityModel
+{
+
+    /// <summary>
+    /// An abstract class that provides a generic property bag to derived classes.
+    /// </summary>
+    public abstract class OpenObject
+    {
```

Comment:
WSTrustMessage is the only derived type. What do you think about moving the Properties property up into that type and getting rid of it as a separate class?

Reply:
do you know why this was in the first place ? I can move(easy)